language/sygus.tex

```diff
@@ -1264,7 +1264,7 @@ \subsection{Asserting Oracle Constraints and Assumptions}
 \subsection{Declaring Oracle Functional Symbols}
 
 \begin{itemize}
-\item $\paren{\oracledeckwd\mbox{ }S\mbox{ } N\mbox{ }\paren{\paren{x_1\mbox{ }\sigma_1} \ldots \paren{x_n\mbox{ }\sigma_n}}\mbox{ }\sigma}$
```

Comment:
The syntax sugar below still refers to x_1, ... x_n.  This is correct, however, can you clarify that x_1, ..., x_n are fresh variables after the explanation (line 1286)?